parsemsgpack.m

```diff
@@ -19,24 +20,21 @@
 
 function [obj, idx] = parse(bytes, idx)
     % masks:
-    b10000000 = 128;
-    b01111111 = 127;
```

Comment:
The masks you removed are unused, I take it?

Reply:
Yes , they are unused.